### PR TITLE
geo-ip-initial 

### PR DESCRIPTION
**Pull request by eliaslopezgt**
# Description

- This plugin should obtain the Client Location Hash from an IP address.

### CODE REVIEW

**Comment by mattiaerre** [comment]
hi @matteofigus,

@eliaslopezgt and I are working on this free geo ip plugin, happy if you can review this PR
thanks,

-M&E

File: index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const fetch = require('node-fetch');
+
+module.exports.register = (options, dependencies, next) => {
+  next();
+};
+
+module.exports.execute = (context) => {
+  const ip = context.requestHeaders['x-forwarded-for'] || context.requestHeaders.remoteAddress;
```
Comment (mattiaerre) [inline index.js:8]: what do you think about this strategy @matteofigus ? will this cover most of the cases?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # oc-free-geo-ip-plugin
+
 OpenComponents FreeGeoIP plugin
```
Comment (mattiaerre) [inline README.md:3]: @eliaslopezgt and I will improve this doc I promise 😅 

// cc @matteofigus

**Review by mattiaerre — COMMENTED** [review]
over to you @matteofigus

**Review by matteofigus — APPROVED** [review]
LGTM

**Comment by mattiaerre** [comment]
thanks @matteofigus you can merge if you like so that I can publish it to npm

// cc @eliaslopezgt

**Pull request merged**